sdk/communication/Azure.Communication.Administration/tests/Infrastructure/PhoneNumberAdministrationClientTestEnvironment.cs

```diff
@@ -8,7 +8,9 @@ namespace Azure.Communication.Administration.Tests
     public class PhoneNumberAdministrationClientTestEnvironment : TestEnvironment
     {
         private const string ConnectionStringEnvironmentVariableName = "COMMUNICATION_CONNECTION_STRING";
+        private const string EndpointStringEnvironmentVariableName = "COMMUNICATION_ENDPOINT_STRING";
 
         public string ConnectionString => GetRecordedVariable(ConnectionStringEnvironmentVariableName);
+        public string EndpointString => GetRecordedVariable(EndpointStringEnvironmentVariableName);
```

Comment:
Ideally, we should only need one config variable. Since end point is contained in the connection string already, can we just parse it from the connection string instead of adding another env variable? It'll simplify env configuration.

Reply:
Hey JP, technically we could, but the Endpoint and Connection string are used for two different ways of authentication. Endpoint can be used for token credential where as connection string is used for access key authentication.

Reply:
Let's do the environment variable cleanup once we finish the `Administration` splitting refactor. I already break the build once 😑